global-dispatcher/cmd/dispatch/base.go

```diff
@@ -47,9 +49,13 @@ func DispatchSessions(ctx context.Context, requestID string) (uint32, error) {
 		return 0, shared.ErrNoCacheClientProvided
 	}
 
-	db, err := database.ClientFromURI(ctx, mongoConnectionString, mongoDatabase)
+	dbClient, err := database.NewPHDClient(dbclient.Config{
+		BaseURL: phdBaseURL,
+		APIKey:  phdAPIKey,
+		Version: dbclient.V1,
+	})
 	if err != nil {
-		return 0, errors.New("error connecting to mongo: " + err.Error())
+		return 0, errors.New("error connecting to phd: " + err.Error())
```

Comment:
Minor note but there is no real "connection" to PHD that happens here (since it is a REST API). The error returned from `NewPHDClient` has to do with validating the config inputs so a more accurate error message would be something indicating the validation of the config struct has failed.